lib/screens/mainScreen.js

```diff
@@ -1,29 +1,58 @@
 import React, { Component } from 'react';
```

Comment:
This is a stateless component, you aren't using `Component` to extend, so remove the import
```suggestion
import React from 'react';
```